veyon.sls

```diff
@@ -5,7 +5,7 @@
 {% set arch = 'win32' %}
 {% endif %}
 
-{% set versions = ['4.0.7', '4.0.6'] %}
+{% set versions = ['4.3.1', '4.3.0'] %}
```

Comment:
Are the older versions no longer available?

Reply:
I did not understand that they are required for upgrade.

Thanks.